scripts/gpg-share.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+ARCHIVE="keys.7z"
```

Comment:
Could you add the version, or maybe a date to avoid having several archives with the same name?

Reply:
> Could you add the version, or maybe a date to avoid having several archives with the same name?
>
sure can :+1: 

i can propose the following
```diff
diff --git a/scripts/gpg-share.sh b/scripts/gpg-share.sh
index 49a424d..6a075fd 100755
--- a/scripts/gpg-share.sh
+++ b/scripts/gpg-share.sh
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 
-ARCHIVE="keys.7z"
-
 
 # TODO: documentation
 log_error () {
@@ -27,6 +25,24 @@ log_info () {
 }
 
 
+# gives a default value to the VERSION file
+[ -z "$VERSION_FILE" ] && VERSION_FILE="package.json"
+
+# get the version
+VERSION=$(jq --raw-output .version "$VERSION_FILE" 2> /dev/null)
+[ -z "$VERSION" ] && {
+  log_error "$VERSION_FILE: No such file or directory"
+  exit 1
+}
+[ "$VERSION" = "null" ] && {
+  log_error "could not find 'version' in '$VERSION_FILE'"
+  exit 1
+}
+
+# build the archive name
+ARCHIVE="keys-${VERSION}.7z"
+
+
 # TODO: documentation
 check_file () {
   [ -z "$1" ] && {
```

Reply:
it simply uses `jq` to parse the `package.json` file

Reply:
You could use `CURRENT_VERSION=$(npm pkg get version | sed 's/"//g')` that does exactly that job in my `bump.sh` script :wink:

Reply:
perfect :ok_hand:

Reply:
should be added in fd81917

Reply:
Seems good to me! @atxr ?